src/pharmpy/tools/modelfit/tool.py

```diff
@@ -3,7 +3,7 @@
 from pharmpy.model import Model
 from pharmpy.workflows import ModelEntry, Task, Workflow, WorkflowBuilder
 
-SupportedPlugin = Literal['nonmem', 'nlmixr']
+SupportedPlugin = Literal['nonmem', 'nlmixr', 'rxode']
```

Comment:
Perhaps it should be called `SupportedPlugins` now, although `Plugin` should perhaps be something else. `SupportedExternalTools` ?